pkg/server/chaosd/network.go

```diff
@@ -511,3 +530,47 @@ func (s *Server) recoverEtcHosts(attack *core.NetworkCommand, uid string) error
 	}
 	return nil
 }
+
+func (s *Server) recoverNICDown(attack *core.NetworkCommand) error {
+	NICUpCommand := fmt.Sprintf("ifconfig %s %s up", attack.Device, attack.IPAddress)
+
+	recoverCmd := exec.Command("bash", "-c", NICUpCommand)
+	if err := recoverCmd.Start(); err != nil {
+		return errors.WithStack(err)
+	}
+
+	return nil
+}
+
+func (s *Server) recoverNICDownScheduled(attack *core.NetworkCommand) error {
```

Comment:
In this way, we need to set the experiment's status to `Destroyed`.

I think it's better to refine the `schedule` framework, and let it recover automatically by the `schedule` framework.

Reply:
We can refine the `schedule` framework in next pr